apps/service_providers/llm_service/runnables.py

```diff
@@ -215,6 +215,7 @@ def _build_chain(self) -> Runnable[dict[str, Any], str]:
             {"input": RunnablePassthrough()}
             | RunnablePassthrough.assign(source_material=RunnableLambda(lambda x: self.source_material))
             | RunnablePassthrough.assign(participant_data=RunnableLambda(lambda x: self.participant_data))
+            | RunnablePassthrough.assign(current_time=RunnableLambda(lambda x: str(timezone.now())))
```

Comment:
FYI: Only the agents will be using the knowlegde of the current time, but I think it would be nice if we can make all our gpt's time aware